JamGame/Assets/Scripts/Level/Executor.cs

```diff
@@ -44,18 +57,34 @@ public class Executor : MonoBehaviour
 
         public UnityEvent ActionEndNotify;
 
-        public SerializedEmployeeConfig TestEmplotyeeConfig;
+        private ConditionsGate conditionsGate;
+
+        public SerializedEmployeeConfig TestEmployeeConfig;
+
+        private IDataProvider<AllEmployeesAtHome> homeConditionProvider;
+        private IDataProvider<AllEmployeesAtMeeting> meetingConditionProvider;
+
+        private void Awake()
+        {
+            conditionsGate = new ConditionsGate(this);
+            if (location is not IDataProvider<AllEmployeesAtHome>)
```

Comment:
You still assume that level is `IDataProvider<AllEmployeesAtHome>`

Reply:
https://github.com/mertwole/miniature-happiness/blob/f2de9ed816fce630ddf1f967ac1c72169abbea8d/JamGame/Assets/Scripts/Level/Boss/Task.cs#L25